pkg/test/testreconciler/simpletest/controller_test.go

```diff
@@ -28,33 +25,6 @@ import (
 	api "sigs.k8s.io/kubebuilder-declarative-pattern/pkg/test/testreconciler/simpletest/v1alpha1"
 )
 
-func TestSimpleReconciler(t *testing.T) {
```

Comment:
split the two test cases in `TestSimpleReconciler` to `with_kubectl_test.go` and `without_kubectl_test.go` so that we can use different tags for each file.

Reply:
sgtm!